include/dlr_common.h

```diff
@@ -5,6 +5,7 @@
 #include <dmlc/logging.h>
 #include <runtime_base.h>
 #include <sys/types.h>
+#include <json.hpp>
```

Comment:
Documentation recommends to include it the following way #include <nlohmann/json.hpp> https://github.com/nlohmann/json#integration.